### ISSUE TITLE
Excessive TCP socket usage for redis

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#1230]
About 3500 ports being used for communication with redis.

And thats not the highest it can reach, thats just a stat I took at a random point during the day on the server : 

```
[root@packetfenceb ~]# netstat -ntu | awk '{print $5}' | grep 127.0.0.1:6379 | wc -l
2535
[root@packetfenceb ~]# netstat -ntu | awk '{print $5}' | grep 127.0.0.1:6380 | wc -l
947
```

Almost all of them are in TIME_WAIT

```
[root@packetfenceb ~]# netstat -ntu | awk '{print $5}' | grep 127.0.0.1:6380 | wc -l && netstat -ntu | grep TIME_WAIT | awk '{print $5}' | grep 127.0.0.1:6380 | wc -l
875
872
```

We need to figure out the high usage of sockets and see if we can lower it by establishing less connections and by making sure we close them when we are done with them as in the environment I analyzed, this accounts for half of the TCP connections on localhost. And this again, in non-peak time.

```
[root@packetfenceb ~]# netstat -ntu | grep TIME_WAIT | awk '{print $5}' | wc -l && netstat -ntu | grep TIME_WAIT | awk '{print $5}' | egrep '127.0.0.1:(6379|6380)' | wc -l
5131
2376
```

**Comment by jrouzierinverse** [comment]
I think I have a good idea why that is happening.

**Comment by jrouzierinverse** [comment]
The redis connection is not shared across each chi namespace.
I work on shared the connection

**Comment by julsemaan** [comment]
Did you have a chance to look at the queuing service as well ?

It was using ~900 connections which were almost all in TIME_WAIT as well

**Comment by jrouzierinverse** [comment]
Fixed by #1233
